views/configs/editcmd.go

```diff
@@ -10,6 +10,73 @@ import (
 	tea "github.com/charmbracelet/bubbletea"
 )
 
+// createConfigInEditorCmd creates a tmp file and opens the editor to create a new config.
```

Comment:
`CreateConfigInEditorCmd` does basically the same as `EditConfigInEditorCmd`, I think we only need one of the two functions, or we can at least extract 90% of the duplicate code to another function.

Reply:
see https://github.com/mosonyi/swarmcli/pull/133/commits/76097f6c7a614915f54d48d39fcbb9a44670eba3